spanner/src/create_backup.php

```diff
@@ -37,7 +37,8 @@
  * @param string $instanceId The Spanner instance ID.
  * @param string $databaseId The Spanner database ID.
  * @param string $backupId The Spanner backup ID.
- * @param string $versionTime The version of the database to backup.
+ * @param string $versionTime The version of the database to backup. Read more
+ * at https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances.backups#Backup.FIELDS.version_time
  */
 function create_backup(string $instanceId, string $databaseId, string $backupId, string $versionTime): void
```

Comment:
The actual change got lost in merges, re-adding that now.

```suggestion
function create_backup(string $instanceId, string $databaseId, string $backupId, string $versionTime = '-1hour'): void
```